package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "0.7.0",
+  "version": "0.8.0",
```

Comment:
Lets just bump the minor version.

Reply:
@maxchehab I think this would be minor bump based on our [versioning policy](https://github.com/workos-inc/workos/wiki/SDK-Release-Process#versioning-policy) since this isn’t a bug fix.